src/Microsoft.FeatureManagement/IFeatureVariantAssignerMetadata.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+namespace Microsoft.FeatureManagement
+{
+    /// <summary>
+    /// Marker interface for feature variant assigners used to assign which variant should be used for a feature.
+    /// </summary>
+    public interface IFeatureVariantAssignerMetadata
```

Comment:
Why do we use marker interface here? Can attribute be used instead?

Reply:
It's used for dependency injection. We have IFeatureVariantAssigner and IContextualFeatureVariantAssigner. We register them in DI under a common interface. IFeatureVariantAssignerMetadata. Attribute would not work for that.